### PR TITLE
CB-9398: Added documentation for WindowsToastCapable to the Windows 

### PR DESCRIPTION
**Pull request by robpaveza**
...docs.

### CODE REVIEW

File: docs/en/edge/guide/platforms/win8/index.md [hunk 1]
```diff
@@ -272,4 +272,18 @@ command-line tools or the SDK in your workflow. The Cordova CLI relies
...
 platform-specific files used by the SDK. If you want to use the SDK to
 modify the project, use the lower-level shell tools as an alternative
-to the CLI.
\ No newline at end of file
+to the CLI.
+
+## Supporting Toasts
+
+Windows requires an app manifest capability declaration in order to support 
+toast notifications.  When using the `cordova-plugin-local-notifications`
```
Comment (nikhilkh) [inline docs/en/edge/guide/platforms/win8/index.md]: I thought a plugin could add preferences

Comment (nikhilkh) [inline docs/en/edge/guide/platforms/win8/index.md]: Ideally, the plugin should do this addition as part of plugin.xml no?

Comment (robpaveza) [inline docs/en/edge/guide/platforms/win8/index.md]: Sure, if a plugin is allowed to modify config.xml, then yes.

Comment (nikhilkh) [inline docs/en/edge/guide/platforms/win8/index.md]: Yes, it is allowed to add preferences.

**Pull request closed without merging**